tests/code/test_search.py

```diff
@@ -29,34 +29,36 @@ def test_searchStudents():
     #tests that the search works
 
     query = 'z'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert searchResults['Zach Neill (neillz)'] == 'Zach Neill (neillz)'
 
 
     #test for last name
     query = 'khatt'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert searchResults['Sreynit Khatt (khatts)'] == 'Sreynit Khatt (khatts)'
 
     #test with non existing username
     query = 'abc'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert len(searchResults) == 0
 
     query = 'its easy as'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert len(searchResults) == 0
 
     query = '123'
-    searchResults = searchUsers(query)
+    searchResults = searchUsers(query, False)
     assert len(searchResults) == 0
 
     #test with multiple users matching query
     with mainDB.atomic() as transaction:
         User.get_or_create(username = 'sawconc', firstName = 'Candace', lastName = 'Sawcon', bnumber = '021556782', isStudent = True)
         query = 'sa'
-        searchResults = searchUsers(query)
+        searchResults = searchUsers(query, False)
+        searchPhoneNumber = searchUsers(query, 'true')
```

Comment:
You can see the mismatch here in datatypes. Is that parameter a string or is it Boolean? Of course, we can mix types in Python and some languages, but this isn't a good case for that, as we have complete control over what is passed.